introduction.tex

```diff
@@ -62,6 +62,9 @@ \subsection{Register Definition Format}
         \hline
         R/W1 & Read/Write. Only writing 1 has an effect.  \\
         \hline
+        R/W1C & Read/Write. Writing every bit high clears the field. Writing
```

Comment:
Do you want this to be Read/Write-1-Clear ?

For multi-bit fields like cmderr, it is actually a bitwise OR, it shouldn't necessarily check that all bits are cleared. For example, you may read a certain error code and write back that error code to "acknowledge" it, without acknowledging other errors.